content/tutorials/webrtc/infrastructure/en/index.html

```diff
@@ -221,66 +212,59 @@ <h3 id="coding-webrtc-for-signaling">Coding WebRTC for signaling</h3>
+<p class="notice tip">The signaling mechanism used to exchange session metadata can also be used to communicate app data. It's just a messaging service!</p>
 
 <h3 id="pushing-messages-from-the-server-to-the-client">Pushing messages from the server to the client</h3>
 
 <p>A message service for signaling needs to be bidirectional: client to server and server to client. Bidirectional communication goes against the HTTP client/server request/response model, but various hacks such as <a href="https://en.wikipedia.org/wiki/Comet_(programming)">long polling</a> have been developed over many years in order to push data from a service running on a web server to a web app running in a browser.</p>
 
-<p>More recently, the <a href="http://www.html5rocks.com/en/tutorials/eventsource/basics/">EventSource API</a> has been <a href="http://caniuse.com/#feat=eventsource">widely implemented</a>. This enables 'server-sent events': data sent from a web server to a browser client via HTTP. There's a simple demo at <a href="http://simpl.info/es">simpl.info/es</a>. EventSource is designed for one way messaging, but it can be used in combination with XHR to build a service for exchanging signaling messages: a signaling service passes on a message from a caller, delivered by XHR request, by pushing it via EventSource to the callee.</p>
+<p>More recently, the <a href="http://www.html5rocks.com/en/tutorials/eventsource/basics/"><code>EventSource</code> API</a> has been <a href="http://caniuse.com/#feat=eventsource">widely implemented</a>. This enables server-sent events&mdash;data sent from a web server to a browser client through HTTP. <code>EventSource</code> is designed for one-way messaging, but it can be used in combination with XHR to build a service for exchanging signaling messages. A signaling service passes a message from a caller, delivered by XHR request, by pushing it through <code>EventSource</code> to the callee.</p>
 
-<p><a href="http://www.html5rocks.com/en/tutorials/websockets/basics/">WebSocket</a> is a more natural solution, designed for full duplex client–server communication (messages can flow in both directions at the same time). One advantage of a signaling service built with pure WebSocket or Server-Sent Events (EventSource) is that the back-end for these APIs can be implemented on a variety of web frameworks common to most web hosting packages, for languages such as PHP, Python and Ruby.</p>
+<p><a href="http://www.html5rocks.com/en/tutorials/websockets/basics/">WebSocket</a> is a more-natural solution, designed for full duplex client–server communication&mdash;messages that can flow in both directions at the same time. One advantage of a signaling service built with pure WebSocket or server-sent events (<code>EventSource</code>) is that the backend for these APIs can be implemented on a variety of web frameworks common to most web-hosting packages for languages, such as PHP, Python, and Ruby.</p>
```

Comment:
Add the comma back after 'packages' and remove the comma after languages. In the updated version the comma makes it sound like these are 'web-hosting packages for languages' whereas they're 'web-hosting packages, for languages such as PHP, Python, and Ruby'.

Reply:
Okay, if that's what you want to say, then I actually think that we need to remove both commas. I'll fix.

Reply:
Okay in that case, we don't want to use either comma. Will fix.